plugins/discourse-ai/assets/javascripts/discourse/components/ai-usage.gjs

```diff
@@ -48,6 +48,7 @@ export default class AiUsage extends Component {
         data: {
           start_date: moment(this.startDate).format("YYYY-MM-DD"),
           end_date: moment(this.endDate).format("YYYY-MM-DD"),
+          timezone: moment.tz.guess(),
```

Comment:
Would it make sense to use `this.currentUser?.user_option?.timezone || moment.tz.guess()` here? Generally speaking, the user's set timezone in their profile is correct and we rely on it in other places (bookmarks, status).